.github/workflows/coverity.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           repository: 'openvinotoolkit/openvino_contrib'
           path: ${{ env.OPENVINO_CONTRIB_REPO }}
           submodules: 'true'
-          ref: 'master'
+          ref: 'releases/2024/4'
```

Comment:
is it possible to generalize using target branch here? in this case we don't have to update this place each release.

Reply:
Need to check if it's possible to use a variable from GitHub context here, if so - yes, it's possible